### ISSUE TITLE
how to custom error log format?

### ISSUE THREAD

**Issue by hzcscut** [openresty/ngx_openresty#112]

**Comment by needcaffeine** [comment]
http://nginx.org/en/docs/http/ngx_http_log_module.html#log_format

**Comment by hzcscut** [comment]
@needcaffeine 
i mean error log, not access log

**Comment by agentzh** [comment]
@needcaffeine No, you can't if you want to use the nginx's builtin error logger since it does not support customisations beyond the custom error message body. You can roll your own error logger in Lua though.

**Comment by keilon** [comment]
I just want to print "one line log" without any builtin format now, how to do this?  Is it recommended to use `io.stderr:write`?
